gtk/gtk.go

```diff
@@ -7472,7 +7472,11 @@ func (v *SelectionData) native() *C.GtkSelectionData {
 	if v == nil {
 		return nil
 	}
-	return v.GtkSelectionData
+
+	// I don't understand why we need this, but we do.
```

Comment:
`v` is already of type `*C.GtkSelectionData`. This cast is not needed.

Reply:
Without this cast, the function would crash the application. The original code also had it.